net/qpid-proton/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+
+name                qpid-proton
+description         A Qpid Proton description goes here
+long_description \
+    Qpid Proton is a high-performance, lightweight messaging library. It can \
+    be used in the widest range of messaging applications, including brokers, \
+    client libraries, routers, bridges, proxies, and more. Proton makes it \
+    trivial to integrate with the AMQP 1.0 ecosystem from any platform, \
+    environment, or language.
+version             0.19.0
+categories          net
+license             Apache-2
+maintainers         @roddiekieley
+platforms           darwin freebsd linux
+homepage            https://qpid.apache.org
+
+fetch.type          git
+git.url             git://github.com/apache/qpid-proton.git
+git.branch          fa80534c87940b44fefd36abff4c5a9a79b99f47
```

Comment:
Use the `github` PortGroup:
```
PortGroup           github 1.0

github.setup        apache qpid-proton fa80534c # use any suitable length of the shasum
version             0.19.0
```

Reply:
@mojca Again thanks for the useful information as this is the first Portfile I have worked on.

0.18.1 was released before the work I've been doing via [PROTON-522](https://issues.apache.org/jira/browse/PROTON-522) was committed via an existing committer there. This means that 0.18.1 will not build on OSX due to a number of issues that I addressed and the shasum I'm using should be when those changes were committed and what I tested here locally and as well was was tested by the committer.

0.19.0 will likely be the next release early in 2018 at which time I'll move to using the version instead of the shasum as the tagged source should then build on OSX.